lnwallet/chanfunding/psbt_assembler_test.go

```diff
@@ -21,14 +21,14 @@ import (
 )
 
 var (
-	localPrivkey                  = []byte{1, 2, 3, 4, 5, 6}
-	remotePrivkey                 = []byte{6, 5, 4, 3, 2, 1}
+	localPrivkey                  = []byte{1, 2, 3, 4, 5, 6} //nolint:unused
```

Comment:
why you deactive this here?

Reply:
Due to the `t.Skipf()` in tests the linter (correctly) assumes the lines of code where these variables are used are never reached, so it complains about them being unused.